torch/csrc/jit/autodiff.cpp

```diff
@@ -0,0 +1,26 @@
+#include "torch/csrc/jit/autodiff.h"
+
+#include "torch/csrc/jit/symbolic_variable.h"
+#include "torch/csrc/utils/functional.h"
+
+namespace torch { namespace jit {
+
+std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_values) {
+  const auto build_sym_grad = [node](const std::vector<SymbolicVariable>& grads) -> std::vector<SymbolicVariable> {
```

Comment:
I think inlining this lambda would be easier to understand,  since it is only used once.

Reply:
I agree, but it would make the code 2x longer, because I couldn't use `return` 😉 I like it more this way, but don't mind inlining it if you'd prefer that way